core/templates/css/oppia.css

```diff
@@ -2572,6 +2572,13 @@ oppia-parameter {
   }
 }
 
+@media(max-width:540px) {
+  .modal-dialog {
```

Comment:
Is this our own CSS or are we trying to override a third party style. If it is something that we added, can you make the selector name a bit more specific. Modal dialog is a very common css selector.

Reply:
yup we are overriding 3rd party style. We have already set style for 768px in oppia.css but not for below 540px due to which the modal does not fit the small screen.
The css of modal-dialog for 768px:

![Screenshot from 2021-08-03 13-28-25](https://user-images.githubusercontent.com/68547101/127979761-8de6fcfa-a4ef-4640-8506-037ef1305395.png)